flutter_flexible_sync/README.md

```diff
@@ -1,52 +1,55 @@
 ![Realm](https://github.com/realm/realm-dart/raw/main/logo.png)
 
 [![License](https://img.shields.io/badge/License-Apache-blue.svg)](LICENSE)
-
-## A simple application using the [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/) Flexible Sync with an [Atlas App Services application](https://www.mongodb.com/docs/atlas/app-services/).
+# Flutter Flexible Sync sample
+A simple application using the [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/) Flexible Sync with an [Atlas App Services application](https://www.mongodb.com/docs/atlas/app-services/).
 This sample demonstrates the usage of Flexible Sync. 
-Writing to a local realm named `db_allTasks.realm` sends the data automatically to the Atlas collection.
-Then the data are downloaded back by the synchronization process to two separate realms
- `db_importantTasks.realm` and  `db_normalTaks.realm` filtered by specific subscription query.
+Writing to a synced realm named `db_allTasks.realm` sends the data automatically to a linked MongoDB collection on Atlas.
+Then the data are downloaded back by the synchronization process to two separate realms,
```

Comment:
```suggestion
```

This seems confusing. Delete it imo

Reply:
I am not sure why we are doing this on this sample. But we should reword this somewhat. 

So perhaps

```
Writing to a synced realm named db_allTasks.realm syncs the data automatically to a linked MongoDB collection on Atlas. 
Then sample demonstrates working with two subscription queries for important tasks and normal tasks in two different realms.

Reply:
OK.